### PR TITLE
Export labels of used tools when the task type is drawing

### LINKED ISSUE #979: Label tools in data export
Looking at the raw classifications, drawing tool markings are just associated with "tool 1", "tool 2" etc. I don't know how the tool number maps to the tool label in the workflow. Can we report the tool label instead of the tool number?

### PR DESCRIPTION
**Pull request by marten**
Fixes #979.

### CODE REVIEW

**Comment by camallen** [comment]
@marten let us know when this is ready for review.

File: spec/lib/formatter/csv/annotation_for_csv_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  it 'adds the task label' do
+    formatted = described_class.new(classification, annotation).to_h
+    expect(formatted["task_label"]).to eq("Draw a circle")
+  end
+
+  it 'adds the tool labels for drawing tasks' do
+    formatted = described_class.new(classification, annotation).to_h
+    expect(formatted["value"][0]["tool_label"]).to eq("Green")
```
Comment (camallen) [inline spec/lib/formatter/csv/annotation_for_csv_spec.rb]: hey if we've got multiple related expectations can you use the new `aggregrate_failures` feature in [rspec](http://rspec.info/blog/2015/06/rspec-3-3-has-been-released/)?

Comment (marten) [inline spec/lib/formatter/csv/annotation_for_csv_spec.rb]: nice, hadn't looked at rspec 3.3 yet.

**Comment by marten** [comment]
@camallen my proposal would be to go ahead with this branch initially (just adding labels inline to the json). I can continue to work on uploading zipfiles with a secondary file describing the project, but it'll be a little more involved as the dump worker logic currently always expects things to be text/csv. 

It occurs to me we could also do a third export download with the project definition, but then we also have to get the frontend involved, and I think it'd be nicer to have the classifications export be a single self-contained and self-describing download.

**Comment by camallen** [comment]
@marten I think we should liase with @ggdhines and @aliburchard to see what they need before implementing something else.

**Pull request merged**